demos/demo-minimal-js/index.js

```diff
@@ -3,115 +3,154 @@
 import CallingExtensions, { Constants } from "@hubspot/calling-extensions-sdk";
 const { errorType } = Constants;
 
-const callback = () => {
-  let rowId = 0;
-  const incomingMsgContainer = document.querySelector("#incomingMsgs");
-  function appendMsg(data, event) {
-    const div = document.createElement("div");
-    rowId += 1;
-    div.innerHTML = `<span>${rowId}: </span><span>${
-      event.type
-    }, ${JSON.stringify(data)}</span>`;
-    incomingMsgContainer.append(div);
+const state = {
+  phoneNumber: "",
+  engagementId: 0,
+};
+
+function disableButtons(ids) {
+  ids.forEach(id => {
+    document.querySelector(`#${id}`).setAttribute("disabled", true);
+  });
+}
+
+function enableButtons(ids) {
+  ids.forEach(id => {
+    document.querySelector(`#${id}`).removeAttribute("disabled");
+  });
+}
+
+function initializeButtons() {
+  enableButtons(["login", "senderror", "resize"]);
+}
+
+function toggleLogin(clickedButtonId) {
```

Comment:
I suggest breaking up this function into `login` and `logout` since `clickedButtonId === "initialize"` will never happen now that we have the `initializeButtons` method.